libnavui-base/src/main/java/com/mapbox/navigation/ui/base/internal/model/route/RouteConstants.kt

```diff
@@ -98,6 +102,9 @@ object RouteConstants {
     @ColorInt
     val ALTERNATE_ROUTE_SEVERE_TRAFFIC_COLOR = Color.parseColor("#B58281")
 
+    @ColorInt
+    val RESTRICTED_ROAD_COLOR = Color.parseColor("#000000")
```

Comment:
Is this color (`#000000`) going to conflict with closures one? https://github.com/mapbox/mapbox-navigation-android/pull/4215#discussion_r605915358

cc @d-prukop

Reply:
We can follow up in a separate ticket. Could you cut a ticket when you have a chance @cafesilencio? 🙏